apps/web/app/blog/page.tsx

```diff
@@ -1,8 +1,25 @@
+import { BlogCard } from '@portfolio/share-ui';
+
 export default async function Index() {
   return (
-    <div className="">
-      <div className="wrapper">
-        <div className="container text-2xl text-rose-600">Welcome to blog!</div>
+    <div className="container mx-auto ">
+      <div className="grid grid-cols-3 gap-4 mt-6">
+        <div className="p-4">
+          <div className="text-xl">Welcome to blog!</div>
+          <div className="mt-2 text-sm">
+            Capturing highlight. Lorem ipsum dolor sit amet, consectetur
+            adipiscing elit.
+          </div>
+        </div>
+
+        {Array.from({ length: 5 }).map((_, i) => (
+          <BlogCard
+            key={i}
+            title="This is Title"
+            subtitle="Capturing highlight. Lorem ipsum dolor sit amet, consectetur adipiscing elit. "
+            imgUrl="https://source.unsplash.com/random"
+          />
+        ))}
```

Comment:
it would be a better example if we could create a list of objects with id, title, subtitle, and URL so that we don't have to use index as key.